src/components/Reports/GoalCalculator/SummaryReport/Steps/PresentingYourGoalStep/GoalApplicationButtonGroup.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import { Box, Button, CircularProgress } from '@mui/material';
+import { useSnackbar } from 'notistack';
+import { useTranslation } from 'react-i18next';
+import { useUpdateAccountPreferencesMutation } from 'src/components/Settings/preferences/accordions/UpdateAccountPreferences.generated';
+import { useAccountListId } from 'src/hooks/useAccountListId';
+import { Goal } from '../../MpdGoal/MpdGoalTable';
+import { useGoalLineItems } from '../../MpdGoal/useGoalLineItems';
+
+interface GoalApplicationButtonGroupProps {
+  goal: Goal;
+}
+
+export const GoalApplicationButtonGroup: React.FC<
+  GoalApplicationButtonGroupProps
+> = ({ goal }) => {
+  const { t } = useTranslation();
+  const { enqueueSnackbar } = useSnackbar();
+  const { overallTotal } = useGoalLineItems(goal);
+  const [updateAccountPreferences, { loading }] =
+    useUpdateAccountPreferencesMutation();
+  const accountListId = useAccountListId() || '';
+  const [buttonsHidden, setButtonsHidden] = useState(false);
+
+  const onSave = async () => {
+    await updateAccountPreferences({
+      variables: {
+        input: {
+          id: accountListId,
+          attributes: {
+            id: accountListId,
+            settings: { monthlyGoal: overallTotal },
+          },
+        },
+      },
+      onCompleted: () => {
+        enqueueSnackbar(t('Saved successfully.'), {
+          variant: 'success',
+        });
+      },
+      onError: () => {
+        enqueueSnackbar(t('Saving failed.'), {
```

Comment:
We have our client configured to automatically show a snackbar message with any errors that come back from GraphQL. We either need to make this message clearer of what failed to save or remove it.